exercises/leap/LeapSpec.groovy

```diff
@@ -0,0 +1,25 @@
+@Grab('org.spockframework:spock-core:1.0-groovy-2.4')
+import spock.lang.*
+
+class LeapSpec extends Specification {
+
+    def 'a year not divisible by 4 is not a leap year'() {
+        expect: new Year(2015).isLeapYear() == false
```

Comment:
> I don't think that either one gives the learner a good cue as to what might be a next step. I'm not exactly sure how to make this better at the moment, but I'll give it some more thought. 

Cool, I don't have any immediate thoughts, but I think it might be something we should think about in general on all the language tracks. Philosophically speaking :)